apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/frist/FristWrapper.tsx

```diff
@@ -1,13 +1,13 @@
 import { isBefore } from 'date-fns'
 import { formaterDato } from '~utils/formatering/dato'
-import classnames from 'classnames'
+import styled from 'styled-components'
+
+const FristSpan = styled.span<{ fristHarPassert: boolean }>`
+  color: ${(p) => p.fristHarPassert && 'var(--a-text-danger)'};
+`
 
 export const FristWrapper = ({ dato }: { dato?: string }) => {
   const fristHarPassert = !!dato && isBefore(new Date(dato), new Date())
 
-  return (
-    <span className={classnames({ 'navds-error-message': fristHarPassert })}>
-      {dato ? formaterDato(dato) : 'Ingen frist'}
-    </span>
-  )
+  return <FristSpan fristHarPassert={fristHarPassert}>{dato ? formaterDato(dato) : 'Ingen frist'}</FristSpan>
```

Comment:
Bare bytte ut denne med en <ErrorMessage>? ref https://aksel.nav.no/komponenter/core/typography#e5a20d9d3128

Reply:
Du må ha en som _ikke_ er errormessage også da, men jeg tenker det gir mening å unngå styled components for denne

Reply:
Tar det fortløpende. Alt tilknyttet frist burde skrives om siden det er rimelig rotete.